### PR TITLE
xfail test bundle create, read, delete workflow

### PR DESCRIPTION
**Pull request by janisz**
## Corresponding DC/OS tickets (required)

  - [D2IQ-65527](https://jira.d2iq.com/browse/D2IQ-65527) Move ip-detect and fault-domain scripts to dcos-config-windows

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -320,8 +320,9 @@ def test_dcos_diagnostics_report(dcos_api_session):
         assert len(report_response['Nodes']) > 0
 
 
-@pytest.mark.parametrize('use_legacy_api',
-                         [False, pytest.param(True, marks=pytest.mark.xfail("config.getoption('--windows-only')"))])
+@pytest.mark.parametrize('use_legacy_api', [
+    pytest.param(False, marks=pytest.mark.xfail("config.getoption('--windows-only')"), reason="D2IQ-65964"),
```
Comment (jeschkies) [inline packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: Please mark it as `strict`

Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: it's strict and premissive

Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: I don't know how to do that. I agree the best will be to xfail only on EE but I think it's fine for now.

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required CI change only

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @jeschkies @DominikDary

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required CI change only

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py (hunk 1 above)
Comment (jeschkies) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: No, I meant make `xfail` fail if the test succeeds. It's not that important as we will tackle it next week.

**Comment by janisz** [comment]
This change was included in #7087

**Pull request closed without merging**